bigchaindb/web/websocket_server.py

```diff
@@ -125,6 +126,9 @@ def websocket_handler(request):
         except RuntimeError as e:
             logger.debug('Websocket exception: %s', str(e))
             break
+        except CancelledError as e:
+            logger.debug('Websocket closed')
```

Comment:
shouldn't this be `info` level instead of `debug`?

Reply:
How much do you care about a disconnected socket?